enforcer/datapath/datapath.go

```diff
@@ -47,10 +46,11 @@ type Datapath struct {
 
 	// Internal structures and caches
 	// Key=ContextId Value=ContainerIP
-	contextTracker cache.DataStore
-	puFromIP       cache.DataStore
-	puFromMark     cache.DataStore
-	puFromPort     cache.DataStore
+	contextTracker   cache.DataStore
+	contextIDTracker cache.DataStore
```

Comment:
1) I looked at this access later and I am confused why we need this data structure - isn't contextID stored in the PU record ? I see the lookup is pu - pointer to context ID; is my understanding correct ? Look at the field called "id" in pucontext.

2) If we really need this data structure should we call it contextIDFromPU which tells whats the key and whats the value.